Data/future/usa/margins/GC.csv

```diff
@@ -33,4 +28,20 @@ date,initial,maintenance
 20160226,4950,4500
 20160627,6050,5500
 20160701,6600,6000
-20160811,6600,6000
+20170113,5940,5400
+20170303,5280,4800
+20170331,4620,4200
+20170601,4345,3950
+20170811,4675,4250
+20170908,4950,4500
+20171110,5390,4900
+20171208,4895,4450
+20171219,4400,4000
+20180109,3850,3500
+20180320,3410,3100
+20181012,3740,3400
+20190621,4070,3700
```

Comment:
Note to anyone else that might be reading this (and my future self):

Even though this is 1.25 years, these were the only changes in that time period.